packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -310,12 +315,84 @@ export function createChildLogger(props?: {
 	return ChildLogger.create(props?.logger, props?.namespace, props?.properties);
 }
 
+/**
+ * Contains sampling strategies for telemetry.
+ *
+ * @remarks This is an object instead of an enum for better comaptibility
+ * https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums
+ */
+export const SamplingStrategy = {
+	/**
+	 * Samples a subset of data from a larger set by selecting every nth event
+	 */
+	SYSTEMATIC: "Systematic",
+	/**
+	 * Samples a subset of data from a larger set using a controlled psuedo-random chance.
+	 */
+	RANDOM: "Random",
+} as const;
+
+/**
+ * This type should be used as the "Enum" for {@link SamplingStrategy}.
+ */
+export type ESamplingStrategy = typeof SamplingStrategy[keyof typeof SamplingStrategy];
```

Comment:
You shouldn't need to name this differently from the `const`. You should be able to just use `SamplingStrategy", which will probably enhance usability a bit.